go.sum

```diff
@@ -48,6 +48,7 @@ google.golang.org/appengine v1.1.0/go.mod h1:EbEs0AVv82hx2wNQdGPgUI5lhzA/G0D9Ywl
 google.golang.org/appengine v1.4.0/go.mod h1:xpcJRLb0r/rnEns0DIKYYv+WjYCduHsrkT7/EB5XEv4=
 google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8/go.mod h1:JiN7NxoALGmiZfu7CAH4rXhgtRTLTxftemlI0sWmxmc=
 google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55/go.mod h1:DMBHOl98Agz4BDEuKkezgsaosCRResVns1a3J2ZsMNc=
+google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 h1:+kGHl1aib/qcwaRi1CbqBZ1rk19r85MNUf8HaBghugY=
```

Comment:
Why there just update the go.sum file but not includes the go.modfile ?

Reply:
because it has referenced an updated commit of gen proto and since gen proto has no releases and go.mod has just referred its name not commit date.
See here https://github.com/googleapis/go-genproto if it makes more sense to you.

Reply:
Thanks, I also have a consider that is I saw the repo https://github.com/googleapis/go-genproto status is currently experimental. How can we clearly avoid the breaking change if the repo upgrade version.

Reply:
This is a peer dependency used by protobuf, I trust the google maintainers that they won't give in any breaking changes with their releases.

Reply:
kindly see the go.mod, or go mod graph to see how dependency are maintained or consumed.